pkgs/by-name/ar/artalk/package.nix

```diff
@@ -31,11 +32,22 @@ buildGoModule rec {
     "-X github.com/ArtalkJS/Artalk/internal/config.Version=${version}"
     "-X github.com/ArtalkJS/Artalk/internal/config.CommitHash=${version}"
   ];
+
+  nativeBuildInputs = [ installShellFiles ];
```

Comment:
This can be moved to [artalk: add completion scripts](https://github.com/NixOS/nixpkgs/pull/315892/commits/4b4833ad91d6b84e1f663d20b3e076b3c98b5a6f)